lewis/devices/julabo/interfaces/julabo_stream_interface_2.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# *********************************************************************
+# plankton - a library for creating hardware device simulators
+# Copyright (C) 2016 European Spallation Source ERIC
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# *********************************************************************
+
+from lewis.adapters.stream import StreamAdapter, Cmd
+
+
+class JulaboStreamInterfaceV2(StreamAdapter):
+    """Julabos can have different commands sets depending on the version number of the hardware.
+
+    This protocol matches that for: FP50-HE (unconfirmed).
+    """
+
+    protocol = "julabo-version-2"
+
+    commands = {
+        Cmd('get_bath_temperature', '^IN_PV_00$'),
+        Cmd('get_external_temperature', '^IN_PV_01$'),
+        Cmd('get_power', '^IN_PV_02$'),
+        Cmd('get_set_point', '^IN_SP_00$'),
+        Cmd('get_high_limit', '^IN_SP_03$'),    # Different from version 1
```

Comment:
Thanks Matt!

Would you prefer we go ahead and merge this as-is for now, or keep it open for you to work on still?

There's no rush to merge it on our end, so it down to which way is more convenient for you.